From bbb7b22c0633fe435c1ac54a1894f82c09686ce2 Mon Sep 17 00:00:00 2001
From: Juffin Alex Varghese <jalex@codeaurora.org>
Date: Mon, 16 Dec 2013 10:28:00 +0530
Subject: [PATCH 579/590] Bluetooth: Replace sprintf with snprintf

Change will replace sprintf with snprintf, since sprintf is a
banned function.

Signed-off-by: Juffin Alex Varghese <jalex@codeaurora.org>
Signed-off-by: Pranav Vashi <neobuddy89@gmail.com>
Signed-off-by: Simarpreet Singh <simar@linux.com>
---
 net/bluetooth/hci_core.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/bluetooth/hci_core.c b/net/bluetooth/hci_core.c
index c8e3330..b52c8ed 100644
--- a/net/bluetooth/hci_core.c
+++ b/net/bluetooth/hci_core.c
@@ -1457,7 +1457,7 @@ int hci_register_dev(struct hci_dev *hdev)
 		head = p; id++;
 	}
 
-	sprintf(hdev->name, "hci%d", id);
+	snprintf(hdev->name, sizeof(hdev->name), "hci%d", id);
 	hdev->id = id;
 	list_add(&hdev->list, head);
 
-- 
1.7.9.5

